Makefile

```diff
@@ -0,0 +1,17 @@
+all: install install-pipenv test
+
+install:
+	sudo apt-get install postgresql-13 \
+		postgresql-client-13 \
+		libhdf5-serial-dev \
+		libnetcdf-dev \
+		libspatialite-dev \
+		postgresql-12-postgis-3
+
+install-pipenv:
+	pip install pipenv
```

Comment:
A question generally: Why are we using a Python install rather than an `apt` install on the system? (I use the `apt` install.)

Reply:
Either one works, I can change this to the `apt` install to remove `pip` from the equation.